internal/service/handlers/create_airdrop.go

```diff
@@ -38,7 +38,7 @@ func CreateAirdrop(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	err = Verifier(r).VerifyProof(req.Data.Attributes.ZkProof, zk.WithAddress(req.Data.Attributes.Address))
+	err = Verifier(r).VerifyProof(req.Data.Attributes.ZkProof, zk.WithEventData(req.Data.Attributes.Address))
```

Comment:
Let's do either of these:
- add WithRarimoAddress to kit with built-in validation
- perform validation in requests.NewCreateAirdrop